enterprise/server/auth/auth.go

```diff
@@ -34,14 +33,15 @@ import (
 	"google.golang.org/grpc/peer"
 
 	akpb "github.com/buildbuddy-io/buildbuddy/proto/api_key"
+	flagtypes "github.com/buildbuddy-io/buildbuddy/server/util/flagutil/types"
 	burl "github.com/buildbuddy-io/buildbuddy/server/util/url"
 	oidc "github.com/coreos/go-oidc"
 )
 
 var (
 	adminGroupID         = flag.String("auth.admin_group_id", "", "ID of a group whose members can perform actions only accessible to server admins.")
 	enableAnonymousUsage = flag.Bool("auth.enable_anonymous_usage", false, "If true, unauthenticated build uploads will still be allowed but won't be associated with your organization.")
-	oauthProviders       = flagutil.New("auth.oauth_providers", []OauthProvider{}, "The list of oauth providers to use to authenticate.")
+	oauthProviders       = flagtypes.Slice("auth.oauth_providers", []OauthProvider{}, "The list of oauth providers to use to authenticate.")
```

Comment:
did you mean to have this diff in this CL?


(same with other flagtypes changes below)